examples/pytorch/text-generation/run_generation.py

```diff
@@ -238,15 +249,13 @@ def __init__(self, optimized, default):
         self._default = default
 
     def __call__(self, *args, **kwargs):
-        if kwargs["past_key_values"] is None:
-            return self._default(*args, **kwargs)
-        trace_graph_inputs = []
+        if kwargs["past_key_values"] is None and self._default.config.use_cache:
+            kwargs["past_key_values"] = generate_past_key_values(self._default, kwargs["input_ids"].shape[0], 0)
```

Comment:
Is this right? For the jit compilation, it's OK to use empty tensors, but in this case, wouldn't we be passing in tensors of random values as the past key, value values?

Reply:
This is used for the first token generation.  It has no past_key_values in this case, but we still need to use the jit model which utilizes past_key_values in jit.trace input, so we have to pass the empty tensor here, because the passkey seq_len + input_seq = attention mask len.  And I have validated the usage, it works, same usage in huggingface optimum-intel.